src/features/course-main/course-main.tsx

```diff
@@ -1,10 +1,11 @@
+import dayjs from 'dayjs';
```

Comment:
You have re-exported `dayjs` as `dayJS` but using the `dayjs` from the library itself, probably re-export could be removed

Reply:
I see that the tests are droped if you remove `dayJS` re-export. It could be fixed if you extend `isBetween` plugin in the `setup-test.ts` file. 

But in this case i don't really like the fact that we need to duplicate `extend` in 2 places if we want to add new plugin, so it's up to you, you can ignore my suggestion

![image](https://github.com/rolling-scopes/site/assets/114234698/b5e97af5-f8ae-4a30-bd00-10ec0aa5ffa5)

Reply:
I assume there will be additions of other plugins as well, so I'd prefer to keep the extensions in one place, but thanks for suggestion